src/mlpack/core/cv/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+# Define the files we need to compile
+# Anything not in this list will not be compiled into mlpack.
+set(SOURCES
+)
```

Comment:
Either we should add the `metrics/*` files here, or an `add_subdirectory(metrics)` call somewhere in this file so that the new files are properly added to `MLPACK_SOURCES`.  Let me know if I can clarify anything about that.